ibis/backends/dask/tests/test_core.py

```diff
@@ -15,19 +11,14 @@
 from dask.dataframe.utils import tm  # noqa: E402
 
 from ibis.backends.dask import Backend  # noqa: E402
-from ibis.backends.dask.core import execute, is_computable_input  # noqa: E402
+from ibis.backends.dask.core import execute  # noqa: E402
 from ibis.backends.dask.dispatch import (  # noqa: E402
     execute_node,
     post_execute,
     pre_execute,
 )
 
 
-@pytest.mark.parametrize('func', [execute_node, pre_execute, post_execute])
-def test_no_execute_ambiguities(func):
```

Comment:
This will fail if there's an ambiguity due to our strict warnings setting for pytest.